bin/fix/fix-audit.rb

```diff
@@ -0,0 +1,188 @@
+require 'rest-client'
+# we use active_fedora/noid to generate pairtrees from noids
+require 'active_fedora/noid'
+require 'yaml'
+require 'rdf/turtle'
+require File.expand_path('../../../config/environment', __FILE__)
+
+def read_config
+  rails_env = Rails.env
+  config = YAML.load_file("config/fedora.yml")
+  @user = config[rails_env]['user']
+  @password = config[rails_env]['password']
+  @rest = config[rails_env]['url']
+  @base_path = config[rails_env]['base_path']
+end
+
+def patch(path, payload)
+  puts 'Patching ' + path + ': ' + payload
```

Comment:
When the script is running, it does create a file for logging:
@f = File.open('audit-fix.log', 'a+'), and the information worth keeping has been written to this log file with @f.puts. -- @f.puts "Patched #{noid} with #{payload}: #{response}"